docs/user_guide/oracle/collection_masker.md

```diff
@@ -1,8 +1,8 @@
 # Masking a collection
 
-While the collection script do not gather any data or source code, it does contain the name of schemas. If there is a requirement to obfuscate this data, there is an optional masking script included with the scripts.
+While the collection script do not gather any data or source code, it does contain the name of schemas and hostname information. If there is a requirement to obfuscate this data, there is an optional masking script included with the scripts.
 
-This script will create a key file that maps the anonymized schema name to its original name.
+This script will create a key file that maps the anonymized schema attribute to its original name.
 
 **Note** This file should is not sent with the collection script and should not be lost, as it can't be recreated.
```

Comment:
"This file should is not sent with the collection" -> "This file should not be sent with the collection"